src/qml/OverlayFeatureFormDrawer.qml

```diff
@@ -80,9 +80,6 @@ Drawer {
 
     property bool isSaved: false
 
-    digitizingToolbar: overlayFeatureFormDrawer.digitizingToolbar
```

Comment:
What's that for?

Reply:
@m-kuhn , it's something we shouldn't have been doing. It's a circular reference (i.e. overlayFeatureFormDrawer.digitizingToolbar is an alias of this very digitizingToolbar property.